### PR TITLE
Add DB setup and DDL

### PR DESCRIPTION
**Pull request by JadenBalogh**

### CODE REVIEW

**Review by AA666666 — APPROVED** [review]

**Comment by AA666666** [comment]
Looks good thanks guys

**Review by aKrisnanto — APPROVED** [review]
DB Setup will need to be updated to match the requirements document

File: Documentation/Dev/Database Design - Milestone 1.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+### maxMonthlyBudget
+Type: INT
+The amount is rounded to the nearest $100. Will discuss rounding in in next sprint (Nov 10th, 2020).
+
+### genderID
+Type: INT
+References the GenderType table.
+
+## SearchPrefs
```
Comment (Jaredhw) [inline Documentation/Dev/Database Design - Milestone 1.md:38]: This got included in anther pull request and is already approved.

**Review by Jaredhw — APPROVED** [review]

**Review by JadenBalogh — COMMENTED** [review]
DB Setup is good for this week; note that this will be updated next week.

**Pull request merged**